.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,8 @@
+<!--
+
+For news posts, please remember:
+
+- Should this be published on Steam? Please attach an image (see release posts for inspiration / dimensions)
```

Comment:
800x450px is what @2TallTyler recommended/used, presumably could put the specific dimensions here.

Reply:
Just too late, sorry; mind making a PR out of that?

I always just take a previous image, but your idea is better :)